tests/containers/helm.pm

```diff
@@ -16,19 +16,58 @@
 
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use utils qw(script_retry);
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
 use mmapi 'get_current_job_id';
+use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::k8s;
 
 sub run {
-    my ($self) = @_;
+    my ($self, $run_args) = @_;
+
+    my $provider_name = get_var('PUBLIC_CLOUD_SERVICE', 'k3s');
```

Comment:
```suggestion
    my $provider_name = get_required_var('PUBLIC_CLOUD_SERVICE');
```

I would make the choice of the kubernetes flavor here mandatory to avoid confusion or is this too verbose?

Reply:
K3s is not a public cloud service, the idea is that if you don't specify this then we don't use a public cloud kubernetes to test the helm. This is the reason to not use required

Reply:
I'd add a comment before this line. Maybe something like:
```
# We either test various cloud clusters or local k3s
```

Reply:
done, thanks!